.github/workflows/storybook-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Storybook Tests
+on: deployment_status
```

Comment:
when do you want this to run?

Reply:
I originally went with the method to run against the deployed storybook, but I've looked at the docs again and am going to update this to the non-deployed method. I mainly just want to integrate these checks into CI.